### ISSUE TITLE
[Accessibility] Implement aria-expanded label on ContextualMenuItem when menu item has submenu

### ISSUE THREAD

**Issue by lejimsft** [OfficeDev/office-ui-fabric-react#3320]
### Bug Report

- __Package version(s)__: 5.20.1
- __Browser and OS versions__: (fill this out if relevant)

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes)

Requested priority: (High) 

Products/sites affected: (if applicable)

#### Describe the issue:

Narrator does not announce the expand/collapse state of a menu item that has submenu when navigated upon. The fix would be to set the 'aria-expanded' label to menu item
